### ISSUE TITLE
Copy results gives "System.NullReferenceException"

### ISSUE THREAD

**Issue by smfukaya** [dnGrep/dnGrep#688]
Hi,

I'm using dnGrep v3.0.64.0 x64 (Windows 10 Version 10H2 Build 19044.1706 x64) and searching with wildcards I'm able to get the results and copy the file names (More ⇨ Copy file names) to the clipboard, but when I try to Copy results (More ⇨ Copy results) it shows "Something broke down in dnGrep. Check the error log for details:".

![20220613_092424-dnGrep_v3 0 64 0_x64 - error while Copying results](https://user-images.githubusercontent.com/29984055/173353748-821251b7-c7f6-4729-9f41-4b48b251c391.jpg)

See the log error below.

```
<logevent time="2022-06-13 09:16:59.1884" level="Error" logger="dnGREP.WPF.App">
  <message>Unhandled exception caught</message>
  <exception>System.NullReferenceException: Object reference not set to an instance of an object.
   at dnGREP.Common.Utils.GetEOL(String path, Encoding encoding)
   at dnGREP.Common.GrepSearchResult.GetLinesWithContext(Int32 linesBefore, Int32 linesAfter)
   at dnGREP.Common.Utils.GetResultLines(List`1 source)
   at dnGREP.WPF.MainViewModel.CopyResults()
   at MS.Internal.Commands.CommandHelpers.CriticalExecuteCommandSource(ICommandSource commandSource, Boolean userInitiated)
   at System.Windows.Controls.MenuItem.InvokeClickAfterRender(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)</exception>
</logevent>
```

**Comment by smfukaya** [comment]
Hi @doug24!

All found files "1.4.8196.29302.txt" are empty. I think this is what is causing the issue. Even if I delete all of them, but one, and copy the results, I got the error.

![20220614_140232-dnGrep x3 0 64 0_x64 - copying results of empty file gives an error](https://user-images.githubusercontent.com/29984055/173635945-8512cd31-f0fc-4a23-8c28-52443cfba7fc.jpg)

But, if the file has some content, no error is raised when copying the results (check the following animated GIF). I know it doesn't make any sense to copy the results of a empty search string from empty files. TBH, I intended to copy the files names, but accidentally clicked on Copy Results...

![20220614_140937-dnGrep x3 0 64 0_x64 - copying results of non-empty file 'works'](https://user-images.githubusercontent.com/29984055/173636686-d3273979-b060-499f-8537-5bd783d5e654.gif)
